Source/Extensions/Blazorise.DataGrid/_DataGridCellDatePicker.razor

```diff
@@ -0,0 +1,91 @@
+@using Blazorise.Utilities
+@typeparam TItem
+@inherits ComponentBase
+
+@if ( TypeHelper.IsDate( valueType ) )
+{
+    @if ( Column.NativeInputMode )
+    {
+        @dateEditFragment
```

Comment:
Just a small note. Make it uppercase so it matches the naming for numeric component, `(NumericEditFragment)`.